javascript/modules/client/src/internal/client/decoding.ts

```diff
@@ -8,7 +8,7 @@ import {
   decodeUpdateMetadataAction,
   decodeWithdrawActionData,
   getFunctionFragment,
-} from "../encoding/client";
+} from "../encoding";
```

Comment:
How are we importing "decodeXXX" functions from a file called `encoding`?

Reply:
it might make more sense logically, what do you think?

Reply:
I would leave it as it is and if it grows too much, split it